experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -747,9 +747,10 @@ def test_targeting_with_prevent_pref_conflicts_set_prefs(self):
             experiment.targeting,
             (
                 '(browserSettings.update.channel == "release") && '
+                "((experiment.slug in activeExperiments) || "
                 "(!('nimbus.test.string'|preferenceIsUserSet)) && "
                 "(!('nimbus.test.int'|preferenceIsUserSet)) && "
-                "(!('nimbus.test.boolean'|preferenceIsUserSet))"
+                "(!('nimbus.test.boolean'|preferenceIsUserSet)))"
```

Comment:
My favourite part of this PR is that all the existing tests remain unchanged and pass and this test changes to the new expected behaviour.  I love it.